pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -177,7 +178,8 @@ func switchSrcDstNodes(switchOrder bool, src, dst Node) (srcRes, dstRes Node) {
 	return src, dst
 }
 
-func (v *VPCConnectivity) computeCombinedConnectionsPerDirection(isIngressDirection bool, node Node, connectivityRes *ConnectivityResult) {
+func (v *VPCConnectivity) computeCombinedConnectionsPerDirection(isIngressDirection bool, node Node,
+	connectivityRes *ConnectivityResult, allowedConnsCombined GeneralConnectivityMap) {
```

Comment:
Its not C++
do not put the result (allowedConnsCombined) as an input

Reply:
The output (and input) is now the receiver